src/parsy/__init__.py

```diff
@@ -106,11 +106,11 @@ def result(self, res):
         return self >> success(res)
 
     def many(self):
-        return self.times(0, float('inf'))
+        return self.at_least(0)
 
     def times(self, min, max=None):
         # max=None means exactly min
-        # min=max=None means from 0 to infinity
+        # max can also be float('inf')
```

Comment:
`float('inf')` is `math.inf` in modern Python; at least this doc should be updated, not sure about the code - do we keep compatibility with earlier Python 3.x versions?

Reply:
Looks like `math.inf` was only added in 3.5, I wouldn't break 3.3 and 3.4 compat for this. Although we could pull it out as a constant at module level for a performance improvement.